clients/vscode/src/commands/commandPalette.ts

```diff
@@ -16,60 +19,85 @@ export class CommandPalette {
   show() {
     const quickPick: QuickPick<CommandPaletteItem> = window.createQuickPick();
     quickPick.title = "Tabby Command Palette";
+    let items: CommandPaletteItem[] = [];
 
-    const items: CommandPaletteItem[] = [];
+    const disconnectStatues = ["disconnected", "unauthorized"];
+    if (disconnectStatues.includes(this.client.status.current?.status || "")) {
+      items.push(this.itemForStatus());
+    }
 
-    items.push(this.itemForStatus());
+    // Status section
     this.client.status.on("didChange", () => {
       items[0] = this.itemForStatus();
       quickPick.items = items;
     });
```

Comment:
This listener function of status changing will always update the first item.  
But the first item may not be status after this PR.

Reply:
I suggest keeping the first item to show the status